CITATION.cff

```diff
@@ -0,0 +1,70 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.0.0
+title: RS-DAT
+message: This file cites all components of RS-DAT
+type: software
+authors:
+  - given-names: Francesco
+    family-names: Nattino
+    email: f.nattino@esciencecenter.nl
+    affiliation: Netherlands eScience Center
+    orcid: 'https://orcid.org/0000-0003-3286-0139'
+  - given-names: Meiert W.
+    family-names: Grootes
+    email: m.grootes@esciencecenter.nl
+    affiliation: Netherlands eScience Center
+    orcid: 'https://orcid.org/0000-0002-5733-4795'
+  - given-names: Fakhereh (Sarah)
+    family-names: Alidoost
+    affiliation: Netherlands eScience Center
+    orcid: 'https://orcid.org/0000-0001-8407-6472'
+  - given-names: Ou
+    family-names: Ku
+    email: o.ku@esciencecenter.nl
+    affiliation: Netherlands eScience Center
+    orcid: 'https://orcid.org/0000-0002-5373-5209'
+  - given-names: Pranav
+    family-names: Chandramouli
+    email: p.chandramoulI@esciencecenter.nl
+    affiliation: Netherlands eScience Center
+    orcid: 'https://orcid.org/0000-0002-7896-2969'
+identifiers:
+  - type: url
+    value: 'https://github.com/RS-DAT/RS-DAT'
+    description: URL to the GitHub repository for RS-DAT
+  - type: doi
+    value: 10.5281/zenodo.7342602
+    description: >-
+      Persistent identifier for all versions of JupyterDaskonSLURM
+  - type: url
+    value: 'https://github.com/RS-DAT/JupyterDaskOnSRC'
+    description: URL to the GitHub repository for JupyterDaskOnSRC
+  - type: url
+    value: 'https://github.com/RS-DAT/JupyterDaskCloud2Cluster'
+    description: >-
+      URL to the GitHub repository for JupyterDask Cloud to
+      Clusterwith Singularity
+  - type: url
+    value: 'https://github.com/RS-DAT/JupyterDask-Singularity'
+    description: >-
+      URL to the GitHub repository for JupyterDask with
+      Singularity
```

Comment:
Maybe only the available DOIs here? The link to this repository is already available below, and the links to the the other RS-DAT repositories could be find from within the documentation.